afew/MailMover.py

```diff
@@ -47,7 +47,47 @@ def __init__(self, max_age=0, rename = False, dry_run=False):
         self.dry_run = dry_run
         self.rename = rename
 
-    def get_new_name(self, fname, destination):
+    @abstractmethod
```

Comment:
Shouldn't the base class (ignoring the abstract here) provide a generic moving implementation?
Moving mails is the tricky part in this feature. We shouldn't encourage duplicating that effort.